recipes/src/main/resources/quarkus-updates/core/3.7.yaml

```diff
@@ -16,4 +16,23 @@ recipeList:
   - org.openrewrite.maven.UpgradePluginVersion:
       groupId: org.apache.maven.plugins
       artifactId: maven-compiler-plugin
-      newVersion: 3.12.1
+      newVersion: 3.12.0
+  - org.openrewrite.maven.ChangePropertyValue:
+      key: compiler-plugin.version
+      newValue: 3.12.0
```

Comment:
Shouldn't it be 3.12.1?

Reply:
It will be but `3.12.1` is not released yet and I don't want to test things with a plugin version that does not exist. I will update the version once we have it.